Reuse.md

```diff
@@ -4,20 +4,20 @@ The initial moonwalk proposal was primarily focused on how to describe the paths
 
 ## Requirements:
 
-- Advertise a list of available API endpoints
-- Enable those API endpoints to either reuse descriptions or use different API descriptions (potentionally with different description formats)
+- Advertise a list of APIs that share an endpoint
```

Comment:
I really should not have used the word "endpoint".  I wanted to say "Advertise a list of API deployment locations".